internal/provider/provider.go

```diff
@@ -29,17 +30,18 @@ import (
 
 // IncusProviderRemoteModel represents provider's schema remote.
 type IncusProviderRemoteModel struct {
-	Name    types.String `tfsdk:"name"`
-	Address types.String `tfsdk:"address"`
-	Port    types.String `tfsdk:"port"`
-	Token   types.String `tfsdk:"token"`
-	Scheme  types.String `tfsdk:"scheme"`
-	Default types.Bool   `tfsdk:"default"`
+	Name      types.String `tfsdk:"name"`
+	Address   types.String `tfsdk:"address"`
+	Protocol  types.String `tfsdk:"protocol"`
+	Auth_Type types.String `tfsdk:"auth_type"`
```

Comment:
Please follow our naming convention (style and use complete words): `AuthenticationType`